### ISSUE TITLE
Incorrect audio track used

### ISSUE THREAD

**Issue by Jaymn** [TeamNewPipe/NewPipe#9399]
### Checklist

- [X] I am able to reproduce the bug with the [latest version](https://github.com/TeamNewPipe/NewPipe/releases/latest).
- [X] I made sure that there are *no existing issues* - [open](https://github.com/TeamNewPipe/NewPipe/issues) or [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have read the [FAQ](https://newpipe.net/FAQ/) and my problem isn't listed.
- [X] I have taken the time to fill in all the required details. I understand that the bug report will be dismissed otherwise.
- [X] This issue contains only one bug.
- [X] I have read and understood the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md).

### Affected version

0.24.1

### Steps to reproduce the bug

Play video with multiple audio tracks. If a different language is the first one (arabic over English in list) it will use that track by default. 

A Few videos I found this issue with (yes it's only Mr.Beast)
1. https://www.youtube.com/watch?v=kX3nB4PpJko
2. https://youtu.be/iogcY_4xGjo
3. https://youtu.be/65fN_OUawjk

### Expected behavior

The audio should play the videos default language, or users/app default language, or be given an option to change the Audio track.

### Actual behavior

Ends up playing the first listed audio track of the video (Arabic with videos posted above)

### Additional information

This may potentially be a feature request, but wasn't sure exactly, so will post here as a bug for the time being. Feel free to let me know if this should be a FR and I can change it

**Comment by ShareASmile** [comment]
Duplicate of #5854

Search better next time before opening new issues for duplicates as  guidelines checklist bullets point 2  says

**Comment by Jaymn** [comment]
Ah sorry about that. When searching "audio track" nothing related came up. It was only after making it plural that that issue came up. GitHub search on mobile doesn't seem the best.

 Closing the ticket.
